src/collectors/diskspace/diskspace.py

```diff
@@ -146,7 +146,8 @@ def get_file_systems(self):
                      mount_point.startswith('/sys'))):
                     continue
 
-                if '/' in device and mount_point.startswith('/'):
+                if ((('/' in device or device == 'tmpfs') and
```

Comment:
Gotcha, `tmpfs` is still skipped though and won't make it to this line since it is not in `filesystems`